src/Data/Nat/Properties.agda

```diff
@@ -203,6 +203,11 @@ m ≤? n = map′ (≤ᵇ⇒≤ m n) ≤⇒≤ᵇ (T? (m ≤ᵇ n))
 _≥?_ : Decidable _≥_
 _≥?_ = flip _≤?_
 
+≤-total : Total _≤_
+≤-total m n with m ≤? n
+... | yes m≤n = inj₁ m≤n
+... | no  m≰n = inj₂ (≰⇒≥ m≰n)
```

Comment:
Perhaps avoid using `yes` and `no`, cf. https://github.com/agda/agda-stdlib/pull/2365/files#r1566627871

Reply:
Thanks for the shoutout, I've been wondering whether I've ridden that hobby-horse a bit too hard, but I do like the new, lazier versions.